### ISSUE TITLE
[Bug]: TypeError in OCP\\Search\\SearchResultEntry for files search provider

### ISSUE THREAD

**Issue by mischaelschill** [nextcloud/server#44263]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

I'm trying to get search results from the REST API (XML). Searching for a term which should return some results ends in a Response with status 200 but Content-Length 0. Searching for a term which should return no results successfully returns an empty entries list. Everything works if json is requested with "Accept: application/json".

### Steps to reproduce

1. Call curl "https://baseurl/ocs/v2.php/search/providers/files/search?term=helden" (a term which should return results)
2. Receive a 200 Response with Content-Length 0
3. Find errors in nextcloud.log

### Expected behavior

Get a list of files containing the word "helden"

### Installation method

Community Docker image

### Nextcloud Server version

28

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
{
    "system": {
        "htaccess.RewriteBase": "\/",
        "memcache.local": "\\OC\\Memcache\\APCu",
        "apps_paths": [
            {
                "path": "\/var\/www\/html\/apps",
                "url": "\/apps",
                "writable": false
            },
            {
                "path": "\/var\/www\/html\/custom_apps",
                "url": "\/custom_apps",
                "writable": true
            }
        ],
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "trusted_proxies": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "cloud.schill.me"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "pgsql",
        "version": "28.0.3.2",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbtableprefix": "nc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "maintenance": false,
        "maintenance_window_start": 1,
        "installed": true,
        "secret": "***REMOVED SENSITIVE VALUE***",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauthtype": "PLAIN",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpauth": 1,
        "mail_smtpsecure": "tls",
        "loglevel": 0,
        "trashbin_retention_obligation": "auto",
        "updatechecker": false,
        "overwrite.cli.url": "http:\/\/cloud.schill.me",
        "dbport": "5432",
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "overwriteprotocol": "https",
        "default_phone_region": "CH",
        "share_folder": "\/Shared with me",
        "theme": "",
        "app_install_overwrite": [
            "video_converter",
            "workflow_media_converter",
            "richdocuments",
            "files_fulltextsearch_tesseract"
        ]
    },
    "apps": {
        "activity": {
            "types": "filesystem",
            "enabled": "yes",
            "installed_version": "2.20.0"
        },
        "audioplayer": {
            "types": "filesystem",
            "installed_version": "3.4.1",
            "enabled": "yes"
        },
        "backgroundjob": {
            "lastjob": "8"
        },
        "bruteforcesettings": {
            "types": "",
            "installed_version": "1.2.0",
            "enabled": "no"
        },
        "calendar": {
            "installed_version": "4.6.6",
            "types": "",
            "enabled": "yes"
        },
        "circles": {
            "installed_version": "28.0.0-dev",
            "types": "filesystem,dav",
            "enabled": "yes",
            "loopback_tmp_scheme": "http",
            "migration_22": "1",
            "migration_22_1": "1",
            "migration_run": "0",
            "maintenance_run": "0",
            "maintenance_update": "{\"maximum\":3,\"3\":1710717132,\"2\":1710718562,\"1\":1710718562,\"4\":1651802402,\"5\":1651287602}"
        },
        "cloud_federation_api": {
            "types": "filesystem",
            "enabled": "yes",
            "installed_version": "1.11.0"
        },
        "comments": {
            "types": "logging",
            "enabled": "yes",
            "installed_version": "1.18.0"
        },
        "contacts": {
            "types": "dav",
            "enabled": "yes",
            "installed_version": "5.5.3"
        },
        "contactsinteraction": {
            "types": "dav",
            "enabled": "yes",
            "installed_version": "1.9.0"
        },
        "core": {
            "installedat": "1540667007.7877",
            "vendor": "nextcloud",
            "public_files": "files_sharing\/public.php",
            "public_webdav": "dav\/appinfo\/v1\/publicwebdav.php",
            "installed.bundles": "[\"CoreBundle\"]",
            "moveavatarsdone": "yes",
            "scss.variables": "db81cddf52fdb3c8ca1e4c859e214124",
            "previewsCleanedUp": "1",
            "updater.secret.created": "1543742241",
            "enterpriseLogoChecked": "yes",
            "lastupdatedat": "1710670494",
            "theming.variables": "b92d206521717ac032f8aa58d3c7ff2f",
            "lastupdateResult": "[]",
            "files_metadata": "{\"photos-original_date_time\":{\"value\":null,\"type\":\"int\",\"indexed\":true,\"editPermission\":0},\"photos-size\":{\"value\":null,\"type\":\"array\",\"indexed\":false,\"editPermission\":0},\"photos-exif\":{\"value\":null,\"type\":\"array\",\"indexed\":false,\"editPermission\":0},\"photos-ifd0\":{\"value\":null,\"type\":\"array\",\"indexed\":false,\"editPermission\":0},\"photos-gps\":{\"value\":null,\"type\":\"array\",\"indexed\":false,\"editPermission\":0},\"photos-place\":{\"value\":null,\"type\":\"string\",\"indexed\":true,\"editPermission\":0}}",
            "backgroundjobs_mode": "cron",
            "files_metadata_installed": "1",
            "oc.integritycheck.checker": "[]",
            "lastcron": "1710718563"
        },
        "dashboard": {
            "types": "",
            "enabled": "yes",
            "installed_version": "7.8.0"
        },
        "dav": {
            "types": "filesystem",
            "enabled": "yes",
            "buildCalendarSearchIndex": "yes",
            "installed_version": "1.29.1",
            "buildCalendarReminderIndex": "yes",
            "chunks_migrated": "1",
            "regeneratedBirthdayCalendarsForYearFix": "yes"
        },
        "federatedfilesharing": {
            "types": "",
            "enabled": "yes",
            "installed_version": "1.18.0"
        },
        "federation": {
            "types": "authentication",
            "enabled": "yes",
            "installed_version": "1.18.0"
        },
        "files": {
            "types": "filesystem",
            "enabled": "yes",
            "cronjob_scan_files": "500",
            "installed_version": "2.0.0",
            "default_quota": "500 GB"
        },
        "files_external": {
            "types": "filesystem",
            "enabled": "yes",
            "user_mounting_backends": "dav,owncloud,sftp,amazons3,swift,smb,\\OC\\Files\\Storage\\SFTP_Key,\\OC\\Files\\Storage\\SMB_OC",
            "allow_user_mounting": "yes",
            "installed_version": "1.20.0"
        },
        "files_fulltextsearch": {
            "files_local": "1",
            "files_external": "0",
            "installed_version": "28.0.0",
            "types": "filesystem",
            "enabled": "yes",
            "files_group_folders": "0",
            "files_encrypted": "0",
            "files_federated": "0",
            "files_pdf": "1",
            "files_office": "1",
            "files_image": "0",
            "files_audio": "0",
            "files_size": "500"
        },
        "files_fulltextsearch_tesseract": {
            "installed_version": "27.0.0",
            "types": "",
            "enabled": "yes",
            "tesseract_enabled": "1",
            "tesseract_psm": "4",
            "tesseract_pdf_limit": "0",
            "tesseract_lang": "eng, deu",
            "tesseract_pdf": "1"
        },
        "files_pdfviewer": {
            "types": "",
            "enabled": "yes",
            "installed_version": "2.9.0"
        },
        "files_reminders": {
            "installed_version": "1.1.0",
            "types": "",
            "enabled": "yes"
        },
        "files_rightclick": {
            "types": "",
            "enabled": "no",
            "installed_version": "1.6.0"
        },
        "files_sharing": {
            "types": "filesystem",
            "enabled": "yes",
            "installed_version": "1.20.0"
        },
        "files_texteditor": {
            "types": "",
            "installed_version": "2.8.0",
            "enabled": "no"
        },
        "files_trashbin": {
            "enabled": "yes",
            "types": "filesystem,dav",
            "installed_version": "1.18.0"
        },
        "files_versions": {
            "enabled": "yes",
            "types": "filesystem,dav",
            "installed_version": "1.21.0"
        },
        "firstrunwizard": {
            "types": "logging",
            "enabled": "yes",
            "installed_version": "2.17.0"
        },
        "fulltextsearch": {
            "cron_err_reset": "1710354258",
            "installed_version": "28.0.1",
            "types": "",
            "enabled": "yes",
            "search_platform": "OCA\\FullTextSearch_Elasticsearch\\Platform\\ElasticSearchPlatform",
            "app_navigation": "1"
        },
        "fulltextsearch_elasticsearch": {
            "types": "",
            "enabled": "yes",
            "installed_version": "28.0.1",
            "elastic_index": "nextcloud",
            "analyzer_tokenizer": "standard",
            "elastic_host": "http:\/\/nextcloud-elasticsearch:9200"
        },
        "gallery": {
            "types": "",
            "installed_version": "18.4.0",
            "enabled": "no"
        },
        "logreader": {
            "enabled": "yes",
            "levels": "01111",
            "types": "logging",
            "installed_version": "2.13.0"
        },
        "lookup_server_connector": {
            "types": "authentication",
            "enabled": "yes",
            "installed_version": "1.16.0"
        },
        "mail": {
            "types": "",
            "enabled": "yes",
            "installed_version": "3.5.7"
        },
        "maps": {
            "enabled": "yes",
            "installed_version": "1.3.1",
            "types": "filesystem"
        },
        "music": {
            "installed_version": "0.9.4",
            "types": "filesystem",
            "enabled": "no"
        },
        "nextcloud_announcements": {
            "types": "logging",
            "pub_date": "Thu, 24 Oct 2019 00:00:00 +0200",
            "enabled": "yes",
            "installed_version": "1.17.0"
        },
        "notes": {
            "installed_version": "4.9.2",
            "types": "",
            "enabled": "yes"
        },
        "notifications": {
            "types": "logging",
            "enabled": "yes",
            "installed_version": "2.16.0"
        },
        "oauth2": {
            "types": "authentication",
            "enabled": "yes",
            "installed_version": "1.16.3"
        },
        "password_policy": {
            "enabled": "yes",
            "types": "authentication",
            "installed_version": "1.18.0"
        },
        "photos": {
            "enabled": "yes",
            "lastPlaceMappedUser": "simona",
            "lastPlaceMappingDone": "true",
            "types": "dav,authentication",
            "installed_version": "2.4.0"
        },
        "polls": {
            "types": "",
            "enabled": "yes",
            "installed_version": "6.1.6"
        },
        "privacy": {
            "types": "",
            "enabled": "yes",
            "installed_version": "1.12.0"
        },
        "provisioning_api": {
            "types": "prevent_group_restriction",
            "enabled": "yes",
            "installed_version": "1.18.0"
        },
        "recommendations": {
            "types": "",
            "enabled": "yes",
            "installed_version": "2.0.0"
        },
        "related_resources": {
            "installed_version": "1.3.0",
            "types": "",
            "enabled": "yes"
        },
        "richdocuments": {
            "types": "prevent_group_restriction",
            "enabled": "yes",
            "installed_version": "8.3.2",
            "public_wopi_url": "https:\/\/cloud.schill.me",
            "disable_certificate_verification": "yes",
            "wopi_url": "http:\/\/cloud.schill.me\/custom_apps\/richdocumentscode\/proxy.php?req="
        },
        "richdocumentscode": {
            "types": "",
            "installed_version": "23.5.904",
            "enabled": "yes"
        },
        "serverinfo": {
            "types": "",
            "enabled": "yes",
            "cached_count_storages": "22",
            "installed_version": "1.18.0",
            "cached_count_filecache": "814548"
        },
        "settings": {
            "types": "",
            "enabled": "yes",
            "installed_version": "1.10.1"
        },
        "sharebymail": {
            "types": "filesystem",
            "enabled": "yes",
            "installed_version": "1.18.0"
        },
        "spreed": {
            "enabled": "yes",
            "types": "dav,prevent_group_restriction",
            "installed_version": "18.0.5",
            "project_access_invalidated": "1",
            "signaling_token_privkey_es256": "***REMOVED SENSITIVE VALUE***",
            "signaling_token_pubkey_es256": "***REMOVED SENSITIVE VALUE***",
            "has_reference_id": "yes"
        },
        "support": {
            "enabled": "yes",
            "types": "session",
            "SwitchUpdaterServerHasRun": "yes",
            "installed_version": "1.11.0"
        },
        "survey_client": {
            "types": "",
            "enabled": "yes",
            "last_sent": "1710103682",
            "installed_version": "1.16.0",
            "last_report": "{\"id\":\"ocbe63bc93c9\",\"items\":[[\"server\",\"version\",\"28.0.3.2\"],[\"server\",\"code\",\"other\"],[\"server\",\"enable_avatars\",\"yes\"],[\"server\",\"enable_previews\",\"yes\"],[\"server\",\"memcache.local\",\"\\\\OC\\\\Memcache\\\\APCu\"],[\"server\",\"memcache.distributed\",\"none\"],[\"server\",\"asset-pipeline.enabled\",\"no\"],[\"server\",\"filelocking.enabled\",\"yes\"],[\"server\",\"memcache.locking\",\"none\"],[\"server\",\"debug\",\"no\"],[\"server\",\"cron\",\"cron\"],[\"php\",\"version\",\"8.2.16\"],[\"php\",\"memory_limit\",1073741824],[\"php\",\"max_execution_time\",0],[\"php\",\"upload_max_filesize\",25769803776],[\"database\",\"type\",\"pgsql\"],[\"database\",\"version\",\"PostgreSQL 14.11 (Debian 14.11-1.pgdg120+2) on x86_64-pc-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit\"],[\"database\",\"size\",6841426723],[\"apps\",\"bruteforcesettings\",\"disabled\"],[\"apps\",\"files_texteditor\",\"disabled\"],[\"apps\",\"gallery\",\"disabled\"],[\"apps\",\"oauth2\",\"1.16.3\"],[\"apps\",\"twofactor_nextcloud_notification\",\"3.8.0\"],[\"apps\",\"files\",\"2.0.0\"],[\"apps\",\"maps\",\"1.3.1\"],[\"apps\",\"files_external\",\"1.20.0\"],[\"apps\",\"files_reminders\",\"1.1.0\"],[\"apps\",\"password_policy\",\"1.18.0\"],[\"apps\",\"workflow_media_converter\",\"1.9.3\"],[\"apps\",\"video_converter\",\"disabled\"],[\"apps\",\"circles\",\"28.0.0-dev\"],[\"apps\",\"activity\",\"2.20.0\"],[\"apps\",\"updatenotification\",\"1.18.0\"],[\"apps\",\"twofactor_backupcodes\",\"1.17.0\"],[\"apps\",\"notes\",\"4.9.2\"],[\"apps\",\"dav\",\"1.29.1\"],[\"apps\",\"polls\",\"6.1.6\"],[\"apps\",\"related_resources\",\"1.3.0\"],[\"apps\",\"music\",\"disabled\"],[\"apps\",\"user_external\",\"0.4\"],[\"apps\",\"survey_client\",\"1.16.0\"],[\"apps\",\"files_rightclick\",\"disabled\"],[\"apps\",\"text\",\"3.9.1\"],[\"apps\",\"files_sharing\",\"1.20.0\"],[\"apps\",\"user_status\",\"1.8.1\"],[\"apps\",\"viewer\",\"2.2.0\"],[\"apps\",\"settings\",\"1.10.1\"],[\"apps\",\"files_trashbin\",\"1.18.0\"],[\"apps\",\"audioplayer\",\"3.4.1\"],[\"apps\",\"notifications\",\"2.16.0\"],[\"apps\",\"systemtags\",\"1.18.0\"],[\"apps\",\"files_versions\",\"1.21.0\"],[\"apps\",\"logreader\",\"2.13.0\"],[\"apps\",\"weather_status\",\"1.8.0\"],[\"apps\",\"privacy\",\"1.12.0\"],[\"apps\",\"spreed\",\"18.0.4\"],[\"apps\",\"provisioning_api\",\"1.18.0\"],[\"apps\",\"calendar\",\"4.6.6\"],[\"apps\",\"recommendations\",\"2.0.0\"],[\"apps\",\"serverinfo\",\"1.18.0\"],[\"apps\",\"fulltextsearch\",\"28.0.1\"],[\"apps\",\"support\",\"1.11.0\"],[\"apps\",\"sharebymail\",\"1.18.0\"],[\"apps\",\"federation\",\"1.18.0\"],[\"apps\",\"twofactor_email\",\"2.7.4\"],[\"apps\",\"workflowengine\",\"2.10.0\"],[\"apps\",\"lookup_server_connector\",\"1.16.0\"],[\"apps\",\"photos\",\"2.4.0\"],[\"apps\",\"comments\",\"1.18.0\"],[\"apps\",\"firstrunwizard\",\"2.17.0\"],[\"apps\",\"cloud_federation_api\",\"1.11.0\"],[\"apps\",\"nextcloud_announcements\",\"1.17.0\"],[\"apps\",\"theming\",\"2.3.0\"],[\"apps\",\"contactsinteraction\",\"1.9.0\"],[\"apps\",\"dashboard\",\"7.8.0\"],[\"apps\",\"federatedfilesharing\",\"1.18.0\"],[\"apps\",\"files_pdfviewer\",\"2.9.0\"],[\"apps\",\"richdocuments\",\"8.3.2\"],[\"apps\",\"richdocumentscode\",\"disabled\"],[\"apps\",\"twofactor_totp\",\"10.0.0-beta.2\"],[\"apps\",\"contacts\",\"5.5.3\"],[\"apps\",\"mail\",\"3.5.7\"],[\"apps\",\"fulltextsearch_elasticsearch\",\"28.0.0\"],[\"stats\",\"num_files\",810972],[\"stats\",\"num_users\",4],[\"stats\",\"num_storages\",22],[\"stats\",\"num_storages_local\",9],[\"stats\",\"num_storages_home\",4],[\"stats\",\"num_storages_other\",9],[\"stats\",\"num_comments\",216],[\"stats\",\"num_comment_markers\",0],[\"stats\",\"num_systemtags\",0],[\"stats\",\"num_systemtags_mappings\",0],[\"files_sharing\",\"num_shares\",21],[\"files_sharing\",\"num_shares_user\",6],[\"files_sharing\",\"num_shares_groups\",3],[\"files_sharing\",\"num_shares_link\",6],[\"files_sharing\",\"num_shares_link_no_password\",6],[\"files_sharing\",\"num_fed_shares_sent\",0],[\"files_sharing\",\"num_fed_shares_received\",0],[\"files_sharing\",\"permissions_0_17\",2],[\"files_sharing\",\"permissions_3_17\",2],[\"files_sharing\",\"permissions_0_31\",2],[\"files_sharing\",\"permissions_0_19\",2],[\"files_sharing\",\"permissions_2_31\",2],[\"files_sharing\",\"permissions_3_31\",2],[\"files_sharing\",\"permissions_1_31\",1],[\"files_sharing\",\"permissions_1_1\",2],[\"files_sharing\",\"permissions_2_1\",4],[\"files_sharing\",\"permissions_3_15\",2],[\"encryption\",\"enabled\",\"no\"],[\"encryption\",\"default_module\",\"no\"]]}"
        },
        "systemtags": {
            "types": "logging",
            "enabled": "yes",
            "installed_version": "1.18.0"
        },
        "text": {
            "enabled": "yes",
            "types": "dav",
            "installed_version": "3.9.1"
        },
        "theming": {
            "types": "logging",
            "enabled": "yes",
            "installed_version": "2.3.0"
        },
        "twofactor_backupcodes": {
            "types": "",
            "enabled": "yes",
            "installed_version": "1.17.0"
        },
        "twofactor_email": {
            "types": "",
            "installed_version": "2.7.4",
            "enabled": "yes"
        },
        "twofactor_nextcloud_notification": {
            "types": "",
            "enabled": "yes",
            "installed_version": "3.9.0"
        },
        "twofactor_totp": {
            "types": "",
            "enabled": "yes",
            "installed_version": "10.0.0-beta.2"
        },
        "updatenotification": {
            "types": "",
            "enabled": "yes",
            "theming": "1.4.5",
            "bruteforcesettings": "1.2.0",
            "mail": "3.5.7",
            "workflow_media_converter": "1.9.1",
            "core": "28.0.3.2",
            "related_resources": "1.0.4",
            "twofactor_email": "2.7.4",
            "video_converter": "1.0.5",
            "installed_version": "1.18.0",
            "richdocumentscode": "22.5.502",
            "twofactor_totp": "6.4.1",
            "polls": "6.1.6",
            "contacts": "5.5.3",
            "calendar": "4.6.6",
            "update_check_errors": "0",
            "music": "0.9.5",
            "audioplayer": "3.4.1",
            "fulltextsearch": "28.0.1",
            "fulltextsearch_elasticsearch": "28.0.1",
            "twofactor_nextcloud_notification": "3.9.0",
            "richdocuments": "8.3.2",
            "spreed": "18.0.5"
        },
        "user_external": {
            "types": "authentication,prelogin",
            "installed_version": "0.4"
        },
        "user_status": {
            "types": "",
            "enabled": "yes",
            "installed_version": "1.8.1"
        },
        "video_converter": {
            "types": "",
            "installed_version": "1.0.5",
            "enabled": "no"
        },
        "viewer": {
            "types": "",
            "enabled": "yes",
            "installed_version": "2.2.0"
        },
        "weather_status": {
            "types": "",
            "enabled": "yes",
            "installed_version": "1.8.0"
        },
        "workflow_media_converter": {
            "convertMediaInParallel": "yes",
            "installed_version": "1.9.3",
            "types": "",
            "enabled": "yes",
            "threadLimit": "32"
        },
        "workflowengine": {
            "types": "filesystem",
            "enabled": "yes",
            "installed_version": "2.10.0"
        }
    }
}
```

### List of activated Apps

```shell
Enabled:
  - activity: 2.20.0
  - audioplayer: 3.4.1
  - calendar: 4.6.6
  - circles: 28.0.0-dev
  - cloud_federation_api: 1.11.0
  - comments: 1.18.0
  - contacts: 5.5.3
  - contactsinteraction: 1.9.0
  - dashboard: 7.8.0
  - dav: 1.29.1
  - federatedfilesharing: 1.18.0
  - federation: 1.18.0
  - files: 2.0.0
  - files_external: 1.20.0
  - files_fulltextsearch: 28.0.0
  - files_fulltextsearch_tesseract: 27.0.0
  - files_pdfviewer: 2.9.0
  - files_reminders: 1.1.0
  - files_sharing: 1.20.0
  - files_trashbin: 1.18.0
  - files_versions: 1.21.0
  - firstrunwizard: 2.17.0
  - fulltextsearch: 28.0.1
  - fulltextsearch_elasticsearch: 28.0.1
  - logreader: 2.13.0
  - lookup_server_connector: 1.16.0
  - mail: 3.5.7
  - maps: 1.3.1
  - nextcloud_announcements: 1.17.0
  - notes: 4.9.2
  - notifications: 2.16.0
  - oauth2: 1.16.3
  - password_policy: 1.18.0
  - photos: 2.4.0
  - polls: 6.1.6
  - privacy: 1.12.0
  - provisioning_api: 1.18.0
  - recommendations: 2.0.0
  - related_resources: 1.3.0
  - richdocuments: 8.3.2
  - richdocumentscode: 23.5.904
  - serverinfo: 1.18.0
  - settings: 1.10.1
  - sharebymail: 1.18.0
  - spreed: 18.0.5
  - support: 1.11.0
  - survey_client: 1.16.0
  - systemtags: 1.18.0
  - text: 3.9.1
  - theming: 2.3.0
  - twofactor_backupcodes: 1.17.0
  - twofactor_email: 2.7.4
  - twofactor_nextcloud_notification: 3.9.0
  - twofactor_totp: 10.0.0-beta.2
  - updatenotification: 1.18.0
  - user_status: 1.8.1
  - viewer: 2.2.0
  - weather_status: 1.8.0
  - workflow_media_converter: 1.9.3
  - workflowengine: 2.10.0
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Logs

```shell
{"reqId":"aSzI6Vq4VKh8GVESNjB8","level":0,"time":"2024-03-17T23:49:16+00:00","remoteAddr":"172.18.0.1","user":"mischael","app":"no app in context","method":"GET","url":"/ocs/v2.php/search/providers/files/search?term=helden","message":"Unable to find providerId definition","userAgent":"curl/8.6.0","version":"28.0.3.2","data":{"value":"files"}}
{"reqId":"aSzI6Vq4VKh8GVESNjB8","level":0,"time":"2024-03-17T23:49:16+00:00","remoteAddr":"172.18.0.1","user":"mischael","app":"no app in context","method":"GET","url":"/ocs/v2.php/search/providers/files/search?term=helden","message":"Unable to find _route definition","userAgent":"curl/8.6.0","version":"28.0.3.2","data":{"value":"ocs.core.UnifiedSearch.search"}}
{"reqId":"aSzI6Vq4VKh8GVESNjB8","level":3,"time":"2024-03-17T23:49:17+00:00","remoteAddr":"172.18.0.1","user":"mischael","app":"PHP","method":"GET","url":"/ocs/v2.php/search/providers/files/search?term=helden","message":"TypeError: XMLWriter::writeElement(): Argument #2 ($content) must be of type ?string, OCP\\Search\\SearchResultEntry given at /var/www/html/lib/private/AppFramework/OCS/BaseResponse.php#163","userAgent":"curl/8.6.0","version":"28.0.3.2","data":{"app":"PHP"}}
```

**Comment by susnux** [comment]
Fixed on master, for 28 we need that backport.

**Comment by joshtrichards** [comment]
Fixed in #44265
